live-editing/configs/ExportExcelConfigGenerator.ts

```diff
@@ -24,33 +24,33 @@ export class ExportExcelConfigGenerator implements IConfigGenerator {
 
         configs.push(new Config({
             component: 'ExcelExportSample1Component',
-            additionalFiles: ["/src/app/services/export-excel/data/invoiceData.ts"],
+            additionalFiles: ['/src/app/data/invoiceData.ts'],
```

Comment:
Grid Excel Export sample is broken in stackblitz/codesandbox - not finding the data.

Reply:
Thanks for catching that! I forgot that `shortenComponentPath` interferes w/ the import path on live editing. They should now be working properly